templates/vhost.j2

```diff
@@ -2,6 +2,7 @@
 {% if item.server_name_redirect is defined %}
 server {
     listen       {{ item.listen | default('80') }};
+    listen       [::]:{{item.listen | default('80') }};
```

Comment:
I think before merging this, I would like this to be behind a flag (and we can default to `true`), like:

```yaml
nginx_listen_ipv6: true
```

And then this line would become:

```
{% if nginx_listen_ipv6 %}
    listen       [::]:{{item.listen | default('80') }};
{% endif %}
```

(And the same for the other occurence.)

Along with this change, the new variable would need to be in defaults and also added in the README.

Reply:
Will do -- I'll get this done tomorrow